source/diagrams/manage-diagrams/rename-diagrams.txt

```diff
@@ -1,13 +1,63 @@
 .. _rm-rename-diagrams:
 
-===============
-Rename Diagrams
-===============
+================
+Rename a Diagram
+================
 
 .. contents:: On this page
-   :local:
-   :backlinks: none
-   :depth: 1
+   :local: 
+   :backlinks: none 
+   :depth: 1 
    :class: singlecol
 
-Rename diagrams.
\ No newline at end of file
+You can rename diagrams on the :guilabel:`Mapping` screen of Relational
+Migrator. Relational Migrator displays diagram names on the Diagram
+Toolbar. 
+
+About this Task 
+---------------
+
+- You can rename diagrams to customize your diagram properties. 
+- Relational Migrator displays your diagram names on the Diagram Toolbar
+  of the :guilabel:`Mapping` screen and in the :guilabel:`Diagram
+  Properties` pane. 
+
+Steps 
+-----
+
+.. procedure::
+   :style:  connected
+
+   .. step:: Navigate to the diagram you want to rename
+
+      Click the name of the diagram in the Diagram Toolbar to display the
```

Comment:
[optional suggestion] Also for visual instructions, it might help to start the sentence with where users should be looking.
```suggestion
      In the Diagram Toolbar, click the name of the diagram to display the
```

Reply:
Done!